### DISCUSSION TITLE
节点方向(direction)

### DISCUSSION THREAD

**Discussion by Zackin-He** [hizzgdev/jsmind#619] [General]
### 当我不知道节点左右方向时(不设置direction)，如何让其自动左右分布，现在默认是全部在右边

**Comment by hizzgdev** [comment]
这有一些文档可能能解答你的问题：
https://hizzgdev.github.io/jsmind/docs/zh/

我正在准备一些新的文档，你也可以参考一下
https://hizzgdev.github.io/jsmind/docs2/zh/advanced/layout.html
https://hizzgdev.github.io/jsmind/docs2/zh/options/option.mode.html

这个问题并不是 issue ，所以会移到 discussion 里去。
